Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "ArraysOfArrays"
 uuid = "65a8f2f4-9b39-5baf-92e2-a9cc46fdf018"
-version = "0.5.7"
+version = "0.5.8"
```

Comment:
@oschulz v.0.5.9 is the one you want to go for ;) [v0.5.8](https://github.com/JuliaArrays/ArraysOfArrays.jl/commit/fb1717b057cca353072998c109a7c31a8095db8f)

Reply:
Ooops, indeed!